src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Interfaces/IPublicationRepository.cs

```diff
@@ -1,16 +1,17 @@
 #nullable enable
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Threading.Tasks;
 using GovUk.Education.ExploreEducationStatistics.Content.Model;
 
 namespace GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces
 {
     public interface IPublicationRepository
     {
-        Task<List<Publication>> GetAllPublicationsForTopic(Guid topicId);
+        IQueryable<Publication> ListPublicationsForTopic(Guid? topicId = null);
```

Comment:
Given this now returns a `IQueryable`, would be tempted to name this `QueryPublicationsForTopic`